2025/07.md

```diff
@@ -173,6 +173,7 @@ When applicable, use these emoji as a prefix to the agenda item topic.
   He would prefer to present on keep-trailing-zeros before the Measure/Amount presentation.
 - Ruben will be able to attend one day only. It is not important which one.
   He would prefer to present all four proposals in one go, especially, since they are all somewhat related to each other.
+- KG would like the base64 item scheduled after the Temporal normative PR item.
```

Comment:
Can you clarify if you mean immediately after or just some time later?

Reply:
Just some time later. Clarified.